core/loadpoint.go

```diff
@@ -317,6 +323,12 @@ func (lp *LoadPoint) configureChargerType(charger api.Charger) {
 		_ = lp.bus.Subscribe(evChargeStop, ct.StopCharge)
 		lp.chargeTimer = ct
 	}
+
+	// add Active timer for wakeup check
+	at := NewActiveTimer()
+	_ = lp.bus.Subscribe(evChargeCurrent, func(current float64) { at.ActiveTimerHandler(lp.enabled, current) })
```

Comment:
warum reagierst Du auf Strom? was willst Du wirklich?

Reply:
Das Problem ist das der Status sich nicht ändert bis das Auto anfängt zu laden und dann braucht man den WakeUp call nicht mehr.
Ich will wissen wann das Auto laden darf.

Reply:
Wenn im Status B die Ladung freigegeben wird (enable) wechselt der Status auf C sobald das Fahrzeug die Ladebereitschaft signalisiert. Wenn es schläft wechselt der Status nicht auf C sondern verbleibt in Status B + enabled.
Mit einem Übergang von B nach C ist das Fahrzeug also definitiv wach bzw. aufgewacht.

Reply:
hab ich geändert - aber schaut gerne nochmal drüber ob euch da noch was auffällt.